.github/workflows/asqi-container-publish.yaml

```diff
@@ -119,12 +113,11 @@ jobs:
             echo "digest=${DIGEST}" >> "$GITHUB_OUTPUT"
           fi
 
-# activate when public GH repo
-#      - name: Attest
-#        if: startsWith(github.ref, 'refs/tags/')
-#        uses: actions/attest-build-provenance@v3
-#        id: attest
-#        with:
-#          subject-name: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
-#          subject-digest: ${{ steps.push.outputs.digest }}
-#          push-to-registry: true
+      - name: Attest
```

Comment:
fyi: I have not had a chance to test this proper. This was copied from another project.

Reply:
Thanks for the note - will see how it works in our next release